### PR TITLE
Add gsassetrules

### PR DESCRIPTION
**Pull request by joyceseng**
In the Overview/Get Started folder, I added the file gs-asset-rules.md. 
I edited the toc.yml to include this file at the end.
Can't remember for sure if I should be trying to merge this branch to main, or Development (but i did not find Development)

Please check it looks ok! Thanks!!

### CODE REVIEW

File: content/overview/get-started/gs-asset-rules.md [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+1. For the **2. Match:** option, select **letters preceding the delimiter "."**. This allows you to use this as a token in the asset configuration. 
+1. In the **and name it:** field, enter **Region**, and select **Capture**. This adds two rules, one for the Region and another for the "." delimiter. 
+1. Next, **SLTC** is selected. In this case, we know our naming pattern enforces a 4-letter abbreviation for the site, so in **3. Match:** select **the next 4 letters**. This helps ensure we do not match any streams that we do not expect. 
+1. Name this token **Site** and select **Capture**.
+1. Now, **PumpA** is selected. Because we want to distinguish pump streams from other streams, use the slider to select only **Pump**, and select **the next 4 characters of any type**. 
+1. Name this token **Equipment** and select **Capture**.
+1. Finally, just **A** is selected. Use the option **letters until the end of stream name**, and name this token **Id**. 
+1. Select **Capture**.
+
+**Note**: Alternatively, you can bring in tokens by using the  **Stream Metadata** button.
```
Comment (afloresOSI) [inline content/overview/get-started/gs-asset-rules.md:81]:
```suggestion
**Note**: Alternatively, you can bring in tokens by using the **Stream Metadata** button. 

```

File: content/overview/get-started/gs-asset-rules.md [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+
+2. [Extract tokens](#extract-tokens-#2): Assemble token values so you can select the information you want from your streams. 
+
+3. [Map tokens](#map-tokens-#3): Map the tokens to values.
+
+4. [Configure assets](#configure assets-#4): Assign tokens values to asset fields.
+
+5. [Preview assets](#preview-assets-#5): Display a preview of the assets that will be created by this rule. 
+
+For more information about asset rules, see the following topics:
```
Comment (afloresOSI) [inline content/overview/get-started/gs-asset-rules.md:33]:
```suggestion
For more information about asset rules, see [Asset rules](xref:ccAssetRules).

```

Then remove the following bulleted list.

File: content/overview/get-started/gs-asset-rules.md [hunk 3]
```diff
@@ -0,0 +1,137 @@
+---
+uid: gsAssetRules
+---
+
+# Get started with asset rules
+
+Use asset rules to automatically create assets based on streams you find by specifying a stream naming pattern. Asset rules work best if your stream names adhere to a consistent naming convention. For example, the following streams are similarly named, and each contains a timestamped series of measurements for similar pumps:
+
+- **MyData.NorthAmerica.SLTC.PumpA.Temperature**
```
Comment (afloresOSI) [inline content/overview/get-started/gs-asset-rules.md:9]: Add blank lines after every list item, bullet or number. It makes the Zoomin output look better.

**Review by afloresOSI — CHANGES_REQUESTED** [review]
A few minor tweaks. Development would be the preference for merging. Check with Mark to see if there is a way to change the into branch.

File: content/overview/get-started/gs-asset-rules.md (hunk 3 above)
Comment (joyceseng) [reply content/overview/get-started/gs-asset-rules.md:9]: Done - thanks

**Comment by joyceseng** [comment]
Thanks, Angela. I added your 3 suggestions.
I think I’ll just close this PR and create another one that wants to Development. I just could not find the Development branch, previously, but I just saw it…

**Pull request closed without merging**